src/v2i-hub/TelematicBridgePlugin/src/TelematicBridgeMsgWorker.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+#include "PluginLog.h"
+#include <tmx/messages/TmxJ2735.hpp>
+#include "TelematicBridgeException.h"
+#include "jsoncpp/json/json.h"
+#include <boost/algorithm/string.hpp>
+
+using namespace tmx::utils;
+using namespace std;
+using namespace tmx::messages;
+
+namespace TelematicBridge
+{
+    using buffer_structure_t = struct buffer_structure
+    {
+        char *buffer;          // buffer array
+        size_t buffer_size;    // this is really where we will write next.
+        size_t allocated_size; // this is the total size of the buffer.
+    };
+
+    class TelematicBridgeMsgWorker
```

Comment:
Does this object have any state information? Is there a reason we need a class for this or could we just define functions here without a class

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/565/commits/5ca81e0164304caaa46367ebb61763ded5c7607b